### ISSUE TITLE
Are pull requests accepted?

### ISSUE THREAD

**Issue by torolecarte** [sgkoishi/Stellaria#5]

**Comment by sgkoishi** [comment]
Yes, I'll merge PRs, but the plugin itself is kinda broken and not updated. You may wanna try [Dimensions](https://github.com/popstarfreas/Dimensions/).
